src/main/java/org/phyloref/jphyloref/JPhyloRef.java

```diff
@@ -13,27 +13,31 @@
 import org.phyloref.jphyloref.commands.TestCommand;
 
 /**
- * Main class for JPhyloRef. Figures out what the user 
- * wants us to do. 
+ * Main class for JPhyloRef. Contains a list of Commands,
+ * as well as the code for determining which Command to
+ * execute. 
  *
  */
 public class JPhyloRef 
 {
+	/** Version of JPhyloRef */
 	public static final String VERSION = "0.0.1-SNAPSHOT";
```

Comment:
I want to keep the version number in sync with the version in the Maven project model (`pom.xml`), and those [should include "SNAPSHOT"](https://maven.apache.org/guides/getting-started/index.html#What_is_a_SNAPSHOT_version) if they're still in development. We might release something eventually, but it's also possible that JPhyloref might remain permanently in development, since most of the stuff we use it for now could be outsourced to better tools like [Owlery](https://github.com/phenoscape/owlery). Does that make sense? Otherwise, I'm happy to use semantic versioning here and to add '-SNAPSHOT' to the version just in the `pom.xml` file.

Reply:
I'm not necessarily opposed to the SNAPSHOT part. But surely we're past v0.0.1 with this, or are you suggesting we're not?

Reply:
You're right! I've bumped the version number to 0.1 and added a Changelog so we can keep track of version changes (97e0676). I'll tag this commit as v0.1 once I've merged it into master.